tensorflow/lite/kernels/test_delegate_providers.cc

```diff
@@ -29,6 +29,8 @@ constexpr char KernelTestDelegateProviders::kUseSimpleAllocator[];
 
 KernelTestDelegateProviders::KernelTestDelegateProviders()
     : delegate_list_util_(&params_) {
+  TFLITE_TOOLS_CHECK(delegate_list_util_.GetCount() > 0)
```

Comment:
It's possible that the test could run without any delegate providers. So, this check could fail?

Reply:
It is possible to run the test without any delegate provider. It is compulsory, though, to have at least **DefaultExecutionProvider** instance available acting as a "dummy delegate" providing the common parameters and flags. Without it the test fails during runtime based on the missing "help" parameter (as described [here](https://github.com/tensorflow/tensorflow/pull/52110#discussion_r718332050)). I admit, nevertheless, that the error message was rather misleading, so I updated it in the latest commit.